openapi.yaml

```
@@ -16985,6 +17032,346 @@ paths:
+      description: ID of the placement group to look up. Run the [Placement Groups List](#placement-groups-list) operation and store the `id` for the applicable placement group.
+      required: true
+      schema:
+        type: integer
+    post:
+      x-linode-grant: read-write
+      tags:
+      - Placement Groups
+      summary: Placement Group Assign
+      description: |
+        Add one or more compute instances to your placement group. Check out our [example API workflow](/docs/products/compute/compute-instances/guides/placement-groups/#create-a-placement-group) to create a placement group and add compute instances.
+      operationId: placementGroupAddLinode
+      x-linode-cli-action: assignLinode
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_write
+      requestBody:
+        required: true
+        description: >
+          The compute instances you want to add to your placement group.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PlacementGroupComputeInstances'
+      responses:
+        '200':
+          description: >
+            The compute instance was added successfully.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X POST -d '{
+                "linode": [123, 456],
+                "non-compliant": false
+              }' \
+              https://api.linode.com/v4/placement/groups/528/assign
+      - lang: CLI
+        source: >
+          linode-cli placement assignLinode 528 \
+            --linode 123 456 \
+            --non-compliant true
+  /placement/groups/{groupId}/unassign:
+    x-linode-cli-command: placement
+    parameters:
+    - name: groupId
+      in: path
+      description: ID of the placement group to look up. Run the [Placement Groups List](#placement-groups-list) operation and store the `id` for the applicable placement group.
+      required: true
+      schema:
+        type: integer
+    post:
+      x-linode-grant: read-write
+      tags:
+      - Placement Groups
+      summary: Placement Group Unassign
+      description: |
+        Remove one or more compute instances from your placement group.
+      x-linode-cli-action: unassignLinode
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_write
+      requestBody:
+        required: true
+        description: >
+          The compute instances you want to remove from your placement group.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PlacementGroupComputeInstances/properties/linodes'
```

Similar to the update command fix:
```suggestion
            schema:
              type: object
              properties:
                linodes:
                  $ref: '#/components/schemas/PlacementGroupComputeInstances/properties/linodes'
```